solr/solr-ref-guide/src/collapse-and-expand-results.adoc

```diff
@@ -142,6 +142,8 @@ The “expand=true” parameter turns on the ExpandComponent. The ExpandComponen
 
 Inside the expanded section there is a _map_ with each group head pointing to the expanded documents that are within the group. As applications iterate the main collapsed result set, they can access the _expanded_ map to retrieve the expanded groups.
 
+If you would like to have only the number of documents found from the groups in the expanded section returned and not the actual documents, you can set "expand.rows=0". Note that when using "expand.rows=0", score won’t be computed for the expanded section even when requested so maxScore will not be available.
```

Comment:
Thanks for the documentation update. I think this should be part of `expand.rows=0`. Something like this https://github.com/apache/lucene-solr/pull/1334/files#diff-9c5999131702feec07a6b7057fa3bcd7R23

Reply:
Example 
```
[IMPORTANT]
====
When `expand.rows=0`, then only the number of documents found for each expanded value is returned. Hence, scores won't be computed even if requested. `maxScore` is set to 0
====
```

Reply:
Updated. Please let me know if it's still in wrong place